climakitae/threshold_tools.py

```diff
@@ -8,6 +8,9 @@
 import pandas as pd
 import xarray as xr
 from scipy import stats
+import panel as pn
+import param
+import math
 
 import lmoments3 as lm
 from lmoments3 import distr as ldistr
```

Comment:
On this branch in the AE hub, I'm getting an import error when trying to call `import climakitae as ck` due to the import lines here. 

Error message: `AttributeError: module 'scipy.stats._continuous_distns' has no attribute 'frechet_r_gen'`

Reply:
I had this same issue: you just have to pip install lmoments in the notebook cell above the import climakitae as ck line and it should resolve :)

Reply:
@nicolejkeeney 

!pip install git+https://github.com/OpenHydrology/lmoments3.git